nightingale/settings.py

```diff
@@ -54,6 +54,7 @@
     'accounts',
     'reports',
     'snappy',
+    'ona'
```

Comment:
Very minor stylistic thing -- we traditionally add a trailing comma on the last item of Python lists and tuples in order to avoid having to add it later when adding more items.